elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/AggregationDataStoreIntegrationTest.java

```diff
@@ -0,0 +1,361 @@
+                                                selections(
+                                                        field("name", "United States"),
+                                                        field("id", "840")
+                                                )
+                                        )
+                                ),
+                                selections(
+                                        field("highScore", 1000),
+                                        field("overallRating", "Good"),
+                                        field(
+                                                "country",
+                                                selections(
+                                                        field("name", "Hong Kong"),
+                                                        field("id", "344")
+                                                )
+                                        )
+                                )
+                        )
+                )
+        ).toResponse();
+
+        runQueryWithExpectedResult(graphQLRequest, expected);
+    }
+
+    @Test
+    @Disabled
+    public void aggregationComputedMetricTest() throws Exception {
+        String graphQLRequest = document(
+                selection(
+                        field(
+                                "videoGame",
+                                selections(
+                                        field("timeSpent"),
+                                        field("sessions"),
+                                        field("timeSpentPerSession"),
+                                        field("timeSpentPerGame")
+                                )
+                        )
+                )
+        ).toQuery();
+
+        String expected = document(
+                selections(
+                        field(
+                                "videoGame",
+                                selections(
+                                        field("timeSpent", 1400),
+                                        field("sessions", 70),
+                                        field("timeSpentPerSession", 20),
+                                        field("timeSpentPerGame", 14)
+                                )
+                        )
+                )
+        ).toResponse();
+
+        runQueryWithExpectedResult(graphQLRequest, expected);
+    }
+
+    private void create(String query, Map<String, Object> variables) throws IOException {
+        runQuery(toJsonQuery(query, variables));
+    }
+
+    private void runQueryWithExpectedResult(
+            String graphQLQuery,
+            Map<String, Object> variables,
+            String expected
+    ) throws IOException {
+        compareJsonObject(runQuery(graphQLQuery, variables), expected);
+    }
+
+    private void runQueryWithExpectedResult(String graphQLQuery, String expected) throws IOException {
```

Comment:
A lot of this code looks like it was copied from GraphQLIT or something.  Maybe we should refactor this into a common class?

Reply:
GraphQLIT wasn't in the version I was working with, but I can still refactor out these methods for future use if that would be helpful.

Reply:
Sure. You can do it as a separate PR if you want or in this one.